backend/src/main/resources/application-prod.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  config:
+    activate:
+      on-profile: prod
```

Comment:
why specify activate on-profile if this file will be loaded on spring.profiles.active=prod anyway?